microsoft/onefuzz
# Delete NSG if no resources associated with it

## Summary of the Pull Request

_What is this about?_

## PR Checklist
* [X] Applies to work item: #1321 

## Info on Pull Request

_What does this include?_
Perform on timer check for unused NSGs and delete them

## Validation Steps Performed

_How does someone test & validate?_
Deploy OneFuzz, add scaleset to a unique region, and then delete scaleset from the region.
Expected behavior - debug proxy is deleted from the above region, and NSG is deleted as well.
